### ISSUE TITLE
font size

### ISSUE THREAD

**Issue by jeppech** [themondays/Dwarf#5]
First of all, what a nice and smooth theme. Redmine really needed that!

Though, in my personal opinion the font size in the tables is a bit too small.

I tried some changes in the browser and ended up with:

Deleted L65-66 in _tables.scss:

``` css
font-size: 11px;
padding: 6px;
```

This actually makes it alot more pleasing too look at.

**Comment by mtozlu** [comment]
@jeppech Absolutely. Redmine needs a modern looking, clean and professional template which Dwarf suits very well.

IMHO; Font-size and Font-Family should be changeable easily. I have Find&Replaced the Font Family to Google's "Open Sans" and modified Font-Sizes everywhere to get a better look.

**Comment by themondays** [comment]
Hey.
Thanks guys, can you please check my recent commit, I will make things much easier if put sizes to style settings.

Thanks,
Jared.

**Comment by jeppech** [comment]
Hi Jared, that's awesome! Just what I needed :)

Also, haven't been able to find this anywhere, does this theme work for Redmine 3.x?
